### PR TITLE
Feature ap 761

### PR DESCRIPTION
**Pull request by jain-raunak**
New wrapper Generation
https://mailinblue.atlassian.net/browse/AP-761

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# SendinBlue's API v3 Go Library
```
Comment (ekta-slit) [inline README.md:1]: Why readme has been removed? Keep it

File: goWrapper/README.md [hunk 2]
```diff
@@ -0,0 +1,366 @@
+# Go API client for newtest
```
Comment (ekta-slit) [inline goWrapper/README.md:1]: This file should be in root

File: goWrapper/git_push.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```
Comment (ekta-slit) [inline goWrapper/git_push.sh:1]: what is the need of this file?

Comment (jain-raunak) [reply goWrapper/git_push.sh]: It was autogenerated, didn't created any hence removed it.

File: README.md (hunk 1 above)
Comment (jain-raunak) [reply README.md:1]: Path corrected

File: goWrapper/README.md (hunk 2 above)
Comment (jain-raunak) [reply goWrapper/README.md]: Path corrected

**Review by ekta-slit — APPROVED** [review]
LGTM

**Pull request merged**